docs/howtos/integrations/integration-context.md

```diff
@@ -0,0 +1,73 @@
+---
+id: integration-context
+title: Integration Cache
+---
+
+## Overview
+
+Occasionally, you might need to store data between integration commands runs.
+
+A common use-case would be storing API tokens which have expiration time (i.e. JWT).
```

Comment:
Can you please elaborate it a little bit more? Something like:


A common use-case would be storing API tokens which have expiration time (i.e. JWT). Very often JWTs (a.k.a. JSON Web Tokens) are generated through an API call and have a validity of several minutes or hours: in order to avoid re-generating tokens every time a command is executed in Demisto, you can cache them using `integrationContext` and retrieve them until they expire.